### PR TITLE
Change AACT_STATIC_FILE_DIR Variable 

### PR DESCRIPTION
**Pull request by kholtzman5610**
I changed AACT_STATIC_FILE_DIR to Rails.configuration.aact[:static_files_directory] and updated all files I needed to do

- I added the  Rails.configuration.aact[:static_files_directory] in the environments test/production/development
- I removed the mention of AACT_STATIC_FILE_DIR updated with the Rails.configuration.aact[:static_files_directory]
- Visited if the pages worked and folder was created in the right place

### CODE REVIEW

**Review by micronix — COMMENTED** [review]
Looks good, did you run the tests?

**Comment by kholtzman5610** [comment]
@micronix yes I had some later on asked Crystal to help me and I understand how everything works now!

File: config/environments/test.rb [hunk 1]
```diff
@@ -18,4 +18,6 @@
   config.action_view.raise_on_missing_translations = true
   config.action_mailer.default_url_options = { host: "www.example.com" }
   config.active_job.queue_adapter = :inline
+
+  config.aact = {static_files_directory: "public/static/aact-files"}
```
Comment (crystal-wb) [inline config/environments/test.rb:22]: This should be changed for tests. We don't want to store temporary test files in the directory with all the other files.

Change the directory to use the tmp folder 'tmp/aact-files'

**Review by crystal-wb — CHANGES_REQUESTED** [review]
This looks fine overall. But there is an issue with the directory you used for the test enviroment. We want to keep our test files separate from our normal operation files. Please see my comment about this.

**Review by crystal-wb — APPROVED** [review]
Looks good

**Pull request merged**